be/src/Infrastructure/Localization/Foundation.Localization/Resources/DownloadCenterResources.de.resx

```diff
@@ -0,0 +1,171 @@
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="CategoryDeleteDownloadCenterConflict" xml:space="preserve">
+    <value>Löschen Sie das mit der Kategorie verknüpfte Download-Center-Element.</value>
+  </data>
+  <data name="CategoryDeletedSuccessfully" xml:space="preserve">
+    <value>Kategorie erfolgreich gelöscht.</value>
+  </data>
+  <data name="CategoryNotFound" xml:space="preserve">
+    <value>Kategorie nicht gefunden</value>
+  </data>
+  <data name="CategorySavedSuccessfully" xml:space="preserve">
+    <value>Kategorie erfolgreich gespeichert.</value>
+  </data>
+  <data name="DownloadCenterItemDeletedSuccessfully" xml:space="preserve">
+    <value>Download-Center-Kategorie erfolgreich gelöscht.</value>
+  </data>
+  <data name="DownloadCenterItemSavedSuccessfully" xml:space="preserve">
+    <value>Download-Center-Kategorie erfolgreich gespeichert.</value>
+  </data>
+  <data name="DownloadCenterNotFound" xml:space="preserve">
+    <value>Download-Center-Kategorie nicht gefunden.</value>
+  </data>
+  <data name="EditCategory" xml:space="preserve">
+    <value>Kategorie bearbeiten</value>
+  </data>
+  <data name="NavigateToCategoriesLabel" xml:space="preserve">
+    <value>Zurück zu den Kategorien</value>
+  </data>
+  <data name="NavigateToDownloadCenter" xml:space="preserve">
+    <value>Zurück zum Downloadcenter</value>
+  </data>
+  <data name="NewCategory" xml:space="preserve">
+    <value>Kategorie hinzufügen</value>
+  </data>
+  <data name="NewDownloadCenter" xml:space="preserve">
+    <value>Fügen Sie eine Download-Center-Kategorie hinzu</value>
+  </data>
+  <data name="ParentCategory" xml:space="preserve">
+    <value>Eltern-Kategorie</value>
+  </data>
+  <data name="ParentCategoryNotFound" xml:space="preserve">
+    <value>Übergeordnete Kategorie nicht gefunden.</value>
+  </data>
+  <data name="SelectCategory" xml:space="preserve">
+    <value>Kategorie wählen</value>
+  </data>
+  <data name="SubcategoriesDeleteCategoryConflict" xml:space="preserve">
+    <value>Zu dieser Kategorie gehörende Unterkategorien löschen</value>
```

Comment:
**suggestion:** typo. It should be "Zu dieser Kategorie gehörende Unterkategorien löschen.".